src/Room.ts

```diff
@@ -119,9 +119,11 @@ export abstract class Room<T= any> extends EventEmitter {
 
   public setPatchRate( milliseconds: number ): void {
     // clear previous interval in case called setPatchRate more than once
-    if ( this._patchInterval ) { clearInterval(this._patchInterval); }
-
-    this._patchInterval = setInterval( this.broadcastPatch.bind(this), milliseconds );
+    if ( milliseconds === null || milliseconds === 0 ) {
+      clearInterval( this._patchInterval );
```

Comment:
`clearInterval` would be always necessary to prevent multiple intervals from being registered in case `setPatchRate` is called multiple times.